kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -4,14 +4,39 @@
 """
 from __future__ import absolute_import, print_function, unicode_literals
 
+import json
+
 from django import template
-from kolibri.core.hooks import NavigationHook
```

Comment:
Why are we still importing a nagivation hook? What's the difference between the UserNavigationHook that we delted and the NavigationHook that we're including here?